from name_judge import name_judge

try:
    name_judge()
except Exception as e:
    print("An error occurred:", e)